source/configure/configuation-in-mattermost-database.rst

```diff
@@ -14,6 +14,12 @@ Benefits to using this option:
 * Ensure all servers in a High Availability deployment have the same configuration, even when new servers are added to the cluster.
 * Automatically deploy SAML certificates and keys to all servers in the cluster.
 
+Note that once you start using configuration in the database, you can't manually edit the active configuration row. You will either need to do one of:
```

Comment:
Maybe also better to clarify that one "shouldn't" do this, rather than "can't". Because an admin can very well edit some DB rows. We should also clarify why one shouldn't do this, so that the impact is clear.

Reply:
Makes sense, they can actually *edit* but it wouldn't take any effect.